crates/neon-runtime/src/neon.cc

```diff
@@ -537,6 +537,28 @@ extern "C" bool Neon_Mem_SameHandle(v8::Local<v8::Value> v1, v8::Local<v8::Value
   return v1 == v2;
 }
 
+extern "C" void *Neon_Mem_NewPersistent(v8::Local<v8::Value> val) {
+  return static_cast<void*>(new Nan::Persistent<v8::Value>(val));
+}
+
+extern "C" void Neon_Mem_New(v8::Local<v8::Value> *out, void *persistent) {
+  Nan::Persistent<v8::Value>* p = static_cast<Nan::Persistent<v8::Value>*>(persistent);
+  *out = Nan::New(*p);
+}
+
+extern "C" void *Neon_Mem_ClonePersistent(void *persistent) {
+  Nan::Persistent<v8::Value>* original = static_cast<Nan::Persistent<v8::Value>*>(persistent);
+  Nan::Persistent<v8::Value>* copy = new Nan::Persistent<v8::Value>();
+  copy->Reset(*original);
+  return static_cast<void*>(copy);
+}
+
+extern "C" void Neon_Mem_DeletePersistent(void *persistent) {
+  Nan::Persistent<v8::Value>* p = static_cast<Nan::Persistent<v8::Value>*>(persistent);
+  p->Reset();
+  delete p;
```

Comment:
The destructor also calls Reset().  Not harmful, just redundant.

Reply:
Thanks, the documentation does not make this clear.